src/app/modules/vault-filter/vault-filter.module.ts

```diff
@@ -28,8 +30,10 @@ import { VaultFilterComponent } from "./vault-filter.component";
     OrganizationOptionsComponent,
     StatusFilterComponent,
     TypeFilterComponent,
+    GetOrgNameFromIdPipe,
+    OrganizationNameBadgeComponent,
```

Comment:
I'm not sure `VaultFiltersModule` is the right declarer for this. `OrganizationNameBadgeComponent`  isn't used by the anything else in `VaultFiltersModules`, it's used by `IndividualVaultModule`. Do you think that make `IndividualVaultModule` a more fitting home?

If it is placed here for code reuse, I suppose that makes sense, but even then the fact that this component is used for a filter is dictated by the `IndividualVaultComponent` - nothing in the component references a filter. If reuse is a concern, perhaps this get its own module.

Reply:
That's a good point, `IndividualVaultModule` is in fact the better place for this as it's not present if in the organization view. I'll test it and see if it works in there. I have concerns that there may be errors on the org vault page, but if Angular evaluates the ngIf before it looks for the component, it should be fine to move.

Reply:
Okay. I think if that doesn't work there is no issue with creating an `OrganizationNameBadgeModule` (or just a `BadgeModule` if this can be abstracted in such a way). It might seem pedantic, but a reused component is a reused component.

Reply:
The issue I was running into ended up being a circular module import, because I need to import this into `LooseComponentsModule` to use in the CiphersComponent.
Splitting OrganizationNameBadgeComponent into its own module did the trick!